vassal-app/src/main/java/VASSAL/configure/StringEnum.java

```diff
@@ -1,6 +1,4 @@
 /*
- *
- * Copyright (c) 2000-2003 by Rodney Kinney
```

Comment:
Probably shouldn't remove the copyright notices.

Reply:
True